ledger_secure_sdk_sys/src/infos.rs

```diff
@@ -1,3 +1,22 @@
+const fn make_c_string<const N: usize>(s: &str) -> [u8; N] {
+    let mut result = [0u8; N];
+    let mut i = 0;
+    while i != s.len() {
+        result[i] = s.as_bytes()[i];
+        i += 1;
+    }
+    result[i] = '\n' as u8;
+    result
+}
+
+macro_rules! const_cstr {
+    ($name: ident, $section: literal, $in_str: expr) => {
+        #[used]
+        #[link_section = $section]
+        static $name: [u8; $in_str.len() + 1] = make_c_string($in_str);
+    };
+}
+
 const fn const_parse_api_level(x: &str) -> u8 {
```

Comment:
Need `#[cfg(not(target_os = "nanos"))]` to avoid build warning 

```
function `const_parse_api_level` is never usedr_secure_sdk_sys                                                                                                                                           
  --> /home/xchapron/Workspace/LedgerHQ/ledger-device-rust-sdk/ledger_secure_sdk_sys/src/infos.rs:20:10
```

Reply:
Done